CSLabs.Api/Controllers/MaintenanceController.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using CSLabs.Api.Models;
+
+namespace CSLabs.Api.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class MaintenanceController : BaseController
+    {
+        public MaintenanceController(BaseControllerDependencies deps) : base(deps) { }
+
+        [HttpGet]
+        public IActionResult GetMaintenances()
+        {
+            return Ok(DatabaseContext.Maintenances.First());
```

Comment:
@Cliftonz Shouldn't the maintenances by grabbed within a time window similarly to the SystemMessages?

Reply:
Yes